### ISSUE TITLE
loosen the package version for torch 2.0

### ISSUE THREAD

**Issue by imliuruiqi** [iejMac/clip-video-encode#74]
When installing this package through ` pip install clip-video-encode` , it can be observed that the package is compatible with pytorch 2.0 as 
```
clip-video-encode 1.3.0 depends on torch<2 and >=1.7.1
```

**Comment by iejMac** [comment]
@imliuruiqi yeah sounds good. I'm super busy these days, if you could submit a PR I'd be happy to merge it

**Comment by imliuruiqi** [comment]
I check the setup.py and it seems that it will get the the `install_requires` from `requirements.txt` and it turns out that the torch version is fine:
```
torch>=1.7.1,<=2
```
maybe you just need to release a new version with the current requirements

**Comment by iejMac** [comment]
ah yeah, I'm planning on doing a release soon, in the mean time you can just git clone and ```pip install -e .```

**Comment by imliuruiqi** [comment]
unfortunately, I was unable to install it by that way, got error with `ERROR: Failed building wheel for pyarrow` mainly blames to the cmake. But nevermind i will wait the release as i am not in a hurry. 
Thanks a lot for your time and take care.
